public/css/breach-detail.css

```diff
@@ -104,20 +98,135 @@ main.breach-detail {
   margin: 0;
 }
 
-.what-to-do-tips {
+/* recommendations */
+
+.rec-img {
+  min-width: 52px;
+  height: 48px;
+  margin: 0 auto;
+  background-position: top center;
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.rec-pw-1 {
+  background-image: url("/img/recommendation-icons/change-password.svg");
```

Comment:
**TL;DR:** All the image paths look 👍. 

I was compelled to write some crude linter to verify all these CSS file paths.

```js
"use strict";

const exec = require("child_process").exec;
const fs = require("fs").promises;
const path = require("path");
const { promisify } = require("util");

const execp = promisify(exec);
const cmd = `git grep -Eno '/img/recommendation-icons/.*?\\.svg'`;

main(cmd);

async function main(cmd) {
  const { stdout, stderr } = await execp(cmd);
  if (stderr) {
    console.error(stderr);
    process.exit(1);
  }

  const css = stdout
    .toString()
    .split("\n")
    .filter(Boolean)
    .map(line => line.trim());

  for (const row of css) {
    const regexp = /^(?<fileinfo>.*):(?<src>.*?)$/;
    const { fileinfo, src } = regexp.exec(row).groups;
    const svgPath = path.join("./public", src);
    try {
      await fs.stat(svgPath);
    } catch (err) {
      console.error(`[${fileinfo}] Cannot find '${svgPath}'`);
      process.exitCode = 2;
    }
  }
}
```